alacritty/src/cli.rs

```diff
@@ -214,6 +201,38 @@ fn parse_class(input: &str) -> Result<Class, String> {
     }
 }
 
+/// Terminal specific cli options which can be passed to new windows via IPC.
+#[derive(Serialize, Deserialize, StructOpt, Default, Debug, Clone, PartialEq)]
+pub struct TerminalOptions {
+    /// Start the shell in the specified working directory.
+    #[structopt(long)]
+    pub working_directory: Option<PathBuf>,
+
+    /// Remain open after child process exits.
```

Comment:
```suggestion
    /// Remain open after child process exit.
```

Either that or `after the child process exits`. Just to be pedantic.